main.go

```diff
@@ -88,6 +101,33 @@ func main() {
 		os.Exit(1)
 	}
 
+	// check if this is standardized STS workflow via OLM and CCO
+	if common.CCOWorkflow() {
+		setupLog.Info("AWS Role ARN specified by the user, following standardized STS workflow")
+		// ROLEARN env var is set via operator subscription
+		roleARN := os.Getenv("ROLEARN")
+		setupLog.Info("getting role ARN", "role ARN =", roleARN)
+
+		// check if cred request API exists in the cluster before creating a cred request
+		setupLog.Info("Checking if credentialsrequest CRD exists in the cluster")
+		credReqCRDExists, err := DoesCRDExist(CloudCredentialGroupVersion, CloudCredentialsCRDName)
+		if err != nil {
+			setupLog.Error(err, "problem checking the existence of CredentialRequests CRD")
+			os.Exit(1)
+		}
+
+		if credReqCRDExists {
+			// create cred request
+			setupLog.Info(fmt.Sprintf("Creating credentials request for role: %s, and WebIdentityTokenPath: %s", roleARN, WebIdentityTokenPath))
+			if err := CreateCredRequest(roleARN, WebIdentityTokenPath, watchNamespace); err != nil {
+				if !errors.IsAlreadyExists(err) {
+					setupLog.Error(err, "unable to create credRequest")
+					os.Exit(1)
```

Comment:
We never really exit(1) in operators. Would this cause operator pod to crashloop backoff?

Reply:
yes we do exit(1) if the tasks are part of the operator init steps, yes this would cause the operator pod to crashloop backoff